app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1302,12 +1309,32 @@ private Cursor makeStreamCursor(CharSequence streamName)
                 .getDao(Stream.class)
                 .queryRaw(
                         "SELECT rowid _id, * FROM streams WHERE "
+                                + Stream.NAME_FIELD
+                                + " LIKE ? ESCAPE '\\' ORDER BY "
+                                + Stream.NAME_FIELD + " COLLATE NOCASE",
+                        DatabaseHelper.likeEscape(streamName.toString()) + "%")
+                .closeableIterator().getRawResults()).getRawCursor();
+    }
+
+
```

Comment:
Documentation Tip : Add javadoc comments for new functions.
Example : https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/models/Message.java#L512